### PR TITLE
Added support to change or hide label when no matches found

### PR DESCRIPTION
**Pull request by swparkaust**
Changes suggested by moser in a pull request from the older repo: https://github.com/crowdint/rails3-jquery-autocomplete/pull/229

### CODE REVIEW

File: lib/assets/javascripts/autocomplete-rails-uncompressed.js [hunk 1]
```diff
@@ -35,6 +35,10 @@
     var _e = e;
     this.init(_e);
   };
+  jQuery.railsAutocomplete.options = {
+    showNoMatches: 'true',
```
Comment (bigtunacan) [inline lib/assets/javascripts/autocomplete-rails-uncompressed.js]: @swparkaust Is there are a reason you are using a string here rather than a bool?

**Comment by swparkaust** [comment]
@bigtunacan Fixed to use boolean instead.

**Comment by swparkaust** [comment]
@bigtunacan could you re-deploy the updated version to rubygem? It's still pulling the older version.

**Comment by bigtunacan** [comment]
@swparkaust I haven't deployed to rubygems yet.  I'll do a deployment over the weekend.

**Comment by swparkaust** [comment]
@bigtunacan Any chance you can deploy the updated version to rubygems?

**Comment by bigtunacan** [comment]
@swparkaust version number updated to 1.0.1 to follow semantic versioning.  Now pushed.

Thanks.

**Pull request merged**